identifiers/country-ch/municipalities.csv

```diff
@@ -0,0 +1,2203 @@
+id,name
+ocd-division/country:ch/canton:zh/municipality:0001,Aeugst am Albis
```

Comment:
What is the source of the munipality identifiers and are they kept stable?

If they won't necessarily be kept stable then consider using the names instead.

Reply:
The number is assigned by the Federal Statistical Office and is the de facto standard to identify municipalities. The identifiers are stable and only change when municipalities merge.